modell/src/main/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/Vilkårsvurderderingsteg.kt

```diff
@@ -1,101 +1,231 @@
+                        reduserteBeløper = listOf(),
+                        aktiviteter = listOf(),
+                        begrunnelse = it.begrunnelseForTilbakekreving,
+                        // TODO: Spør foreldelsevurderingen
                         foreldet = false,
-                    ),
-                    VurdertVilkårsvurderingsperiodeDto(
-                        periode =
-                            Datoperiode(
-                                fom = LocalDate.now().minusDays(14),
-                                tom = LocalDate.now(),
-                            ),
-                        feilutbetaltBeløp = BigDecimal("0.0"),
-                        hendelsestype = Hendelsestype.SATSER,
-                        reduserteBeløper =
-                            listOf(
-                                RedusertBeløpDto(
-                                    trekk = true,
-                                    beløp = BigDecimal("0.0"),
-                                ),
-                            ),
-                        aktiviteter =
-                            listOf(
-                                AktivitetDto(
-                                    aktivitet = "",
-                                    beløp = BigDecimal("0.0"),
-                                ),
-                            ),
                         vilkårsvurderingsresultatInfo =
-                            VurdertVilkårsvurderingsresultatDto(
-                                vilkårsvurderingsresultat = Vilkårsvurderingsresultat.GOD_TRO,
-                                godTro =
-                                    VurdertGodTroDto(
-                                        beløpErIBehold = true,
-                                        beløpTilbakekreves = BigDecimal("0.0"),
-                                        begrunnelse = "",
-                                    ),
-                                aktsomhet =
-                                    VurdertAktsomhetDto(
-                                        aktsomhet = Aktsomhet.FORSETT,
-                                        ileggRenter = false,
-                                        andelTilbakekreves = BigDecimal("1.0"),
-                                        beløpTilbakekreves = BigDecimal("0.0"),
-                                        begrunnelse = "",
-                                        særligeGrunner =
-                                            listOf(
-                                                VurdertSærligGrunnDto(
-                                                    SærligGrunn.HELT_ELLER_DELVIS_NAVS_FEIL,
-                                                    begrunnelse = "",
+                            it.vurdering.let { vurdering ->
+                                when (vurdering) {
+                                    is Vurdering.FeilaktigeOpplysningerFraBruker ->
+                                        VurdertVilkårsvurderingsresultatDto(
```

Comment:
Binde-s/Fuge-s brukes vel ikke med mer enn to ord? Gjelder flere variabler og klasser her
```suggestion
                                        VurdertVilkårsvurderingResultatDto(
```